lib/child_process.js

```diff
@@ -118,18 +120,26 @@ exports.execFile = function(file /*, args, options, callback*/) {
     env: null
   };
 
-  // Parse the parameters.
+  // Parse the optional positional parameters.
+  var pos = 1;
+  if (pos < arguments.length && Array.isArray(arguments[pos])) {
+    args = arguments[pos++];
+  } else if (pos < arguments.length && arguments[pos] == null) {
+    pos++;
+  }
```

Comment:
Given that we'll need to revisit this anyway later. Let's leave it as is
for now and do any additional cleanups in the next round.
On Sep 2, 2015 10:29 PM, "thefourtheye" <notifications@github.com> wrote:

> In lib/child_process.js
> <https://github.com/nodejs/node/pull/2667#discussion_r38614155>:
>
> > @@ -118,18 +120,26 @@ exports.execFile = function(file /*, args, options, callback*/) {
> >      env: null
> >    };
> >
> > -  // Parse the parameters.
> > +  // Parse the optional positional parameters.
> > +  var pos = 1;
> > +  if (pos < arguments.length && Array.isArray(arguments[pos])) {
> > +    args = arguments[pos++];
> > +  } else if (pos < arguments.length && arguments[pos] == null) {
> > +    pos++;
> > +  }
>
> Right, but it will improve the readability and reduce redundant checks.
> It's just a suggestion I am okay without that as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/2667/files#r38614155>.
>